### DISCUSSION TITLE
License registration valid but no way to update it

### DISCUSSION THREAD

**Discussion by thescrapyarduk** [aclap-dev/video-downloadhelper#1122] [License & Purchase requests]
### What's your issue?

I use Waterfox on Windows 11.

I recently reinstalled Windows 11 to solve a few minor annoyances, I use an x-lite version, different build but still 24H2 and exactly the same browser I used to initially register the add-on.

Registration code valid, but no way for code to be accepted. Box for code is there but not a button to check license just stays on code box when enter pressed to accept code.

I don't have a check license option or any way to validate the license just entered on the waterfox add-on options.

### The 10 first digits (only 10) of your license number or email address

52dfd36b93

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by thescrapyarduk** [comment]
Ignore that message, I posted it anyway in case anybody else is having the same strange issues with Waterfox on Windows 11 systems I have been having the last week or so after a Windows 11 reinstall.

I managed to solve it by choosing show add-on in the sidebar, not just on the toolbar as I do normally for all Waterfox add-ons. 

Choosing use the sidebar showed check license option and all the settings have appeared again, as I also had a problem that settings to change download folder also not available, that had disappeared since the last add-on update IF using the add-on from the toolbar in Waterfox but everything was working and available from the toolbar BEFORE I reinstalled Windows 11.

Not sure why it won't show license or any settings from the toolbar, but works fine from the sidebar option in Waterfox. I might have to check a few more add-ons to see if they show options that are not there from the toolbar.

**Reply by paulrouget** [reply]
> Not sure why it won't show license

You need to scroll down to see the check license button.

**Reply by GrampaWildWilly** [reply]
Never noticed that before.  If you enable `Make the addon popup larger`, the `Check license key` button is visible on the first page of `More settings`.  I've always had the setting enabled so I didn't realize it would affect that particular thing.
